components/prism-opencl.js

```diff
@@ -0,0 +1,49 @@
|barrier|bitselect|cbrt|ceil|clamp|clz|copies|copysign|cos|cosh|cospi|cross|degrees|distance|dot|endian|erf|erfc|exp|exp10|exp2|expm1|fabs|fast_distance|fast_length|fast_normalize|fdim|floor|fma|fmax|fmin|fract|frexp|fro|from|get_global_id|get_global_offset|get_global_size|get_group_id|get_image_channel_data_type|get_image_channel_order|get_image_depth|get_image_dim|get_image_height|get_image_width|get_local_id|get_local_size|get_num_groups|get_work_dim|hadd|half_cos|half_divide|half_exp|half_exp10|half_exp2|half_log|half_log10|half_log2|half_powr|half_recip|half_rsqrt|half_sin|half_sqrt|half_tan|hypot|ilogb|isequal|isfinite|isgreater|isgreaterequal|isinf|isless|islessequal|islessgreater|isnan|isnormal|isnotequal|isordered|isunordered|ldexp|length|lgamma|lgamma_r|log|log10|log1p|log2|logb|mad|mad24|mad_hi|mad_sat|max|mem|mem_fence|min|mix|modf|mul24|mul_hi|nan|native_cos|native_divide|native_exp|native_exp10|native_exp2|native_log|native_log10|native_log2|native_powr|native_recip|native_rsqrt|native_sin|native_sqrt|native_tan|nextafter|normalize|pow|pown|powr|prefetch|radians|read_imagef|read_imageh|read_imagei|read_imageui|read_mem_fence|remainder|remquo|reqd_work_group_size|rhadd|rint|rootn|rotate|round|rsqrt|select|shuffle|shuffle2|sign|signbit|sin|sincos|sinh|sinpi|smoothstep|sqrt|step|sub_sat|tan|tanh|tanpi|tgamma|to|trunc|upsample|vec_step|vec_type_hint|vload16|vload2|vload3|vload4|vload8|vload_half|vload_half16|vload_half2|vload_half3|vload_half4|vload_half8|vloada_half16|vloada_half16_rte|vloada_half16_rtp|vloada_half16_rtz|vloada_half2|vloada_half2_rte|vloada_half2_rtp|vloada_half2_rtz|vloada_half3|vloada_half3_rte|vloada_half3_rtp|vloada_half3_rtz|vloada_half4|vloada_half4_rte|vloada_half4_rtp|vloada_half4_rtz|vloada_half8|vloada_half8_rte|vloada_half8_rtp|vloada_half8_rtz|vstore16|vstore2|vstore3|vstore4|vstore8|vstore_half|vstore_half16|vstore_half16_rte|vstore_half16_rtn|vstore_half16_rtp|vstore_half16_rtz|vstore_half2|vstore_half2_rte|vstore_half2_rtn|vstore_half2_rtp|vstore_half2_rtz|vstore_half3|vstore_half3_rte|vstore_half3_rtn|vstore_half3_rtp|vstore_half3_rtz|vstore_half4|vstore_half4_rte|vstore_half4_rtn|vstore_half4_rtp|vstore_half4_rtz|vstore_half8|vstore_half8_rte|vstore_half8_rtn|vstore_half8_rtp|vstore_half8_rtz|vstore_half_rte|vstore_half_rtn|vstore_half_rtp|vstore_half_rtz|vstore_rte|vstore_rtn|vstore_rtp|vstore_rtz|vstorea_half16|vstorea_half16_rte|vstorea_half16_rtn|vstorea_half16_rtp|vstorea_half16_rtz|vstorea_half2|vstorea_half2_rte|vstorea_half2_rtn|vstorea_half2_rtp|vstorea_half2_rtz|vstorea_half3|vstorea_half3_rte|vstorea_half3_rtn|vstorea_half3_rtp|vstorea_half3_rtz|vstorea_half4|vstorea_half4_rte|vstorea_half4_rtn|vstorea_half4_rtp|vstorea_half4_rtz|vstorea_half8|vstorea_half8_rte|vstorea_half8_rtn|vstorea_half8_rtp|vstorea_half8_rtz|vstorea_half_rte|vstorea_half_rtn|vstorea_half_rtp|vstorea_half_rtz|wait_group_events|work_group_size_hint|write_imagef|write_imageh|write_imagei|write_imageui|write_mem_fence)\b/,
```

Comment:
Same comment here. I can see a lot of common prefixes that could help combining those alternatives into a shorter regexp. (`native_`, `read_`, `vstore_`, etc.)